tests/device_tests/test_msg_resetdevice_supershamir.py

```diff
@@ -0,0 +1,232 @@
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),
+                    proto.ButtonRequest(code=B.Other),
+                    proto.ButtonRequest(code=B.Success),  # show seeds ends here
+                    proto.ButtonRequest(code=B.Success),
+                    proto.Success(),
+                    proto.Features(),
+                ]
+            )
+            self.client.set_input_flow(input_flow)
+
+            # No PIN, no passphrase, don't display random
+            device.reset(
+                self.client,
+                display_random=False,
+                strength=strength,
+                passphrase_protection=False,
+                pin_protection=False,
+                label="test",
+                language="english",
+                backup_type=ResetDeviceBackupType.Slip39_Multiple_Groups,
+            )
+
+        # Check if device is properly initialized
+        resp = self.client.call_raw(proto.Initialize())
```

Comment:
let's not `call_raw` here, examining `device.features` should be good enough

Reply:
when I try to use device.features I get

`AttributeError: module 'trezorlib.device' has no attribute 'features'`

but self.client.features seems to work, is it equivalent?

Reply:
i made a mistake in the original post, it's supposed to be `self.client.features`. Please use that.

Reply:
ok, done